Mods/ArchipelagoCore.SC2Mod/Base.SC2Data/GameData/GameData.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Catalog>
+    <CGame id="Dflt">

```

Comment:
Are you overriding an existing enity? If so, add the `<!-- Override -->` comment

Reply:
Adding to it, from what I understand `Dflt` is just "defauilt"